superset/models.py

```diff
@@ -1661,7 +1661,11 @@ def get_druid_version(self):
         endpoint = (
             "http://{obj.coordinator_host}:{obj.coordinator_port}/status"
         ).format(obj=self)
-        return json.loads(requests.get(endpoint).text)['version']
+        ver = json.loads(requests.get(endpoint).text)['version']
```

Comment:
My programmer skills are quite weak and my knowledge about superset architecture is even more weak. Can you point me on how do I do that? Or maybe you can help me by coding this.